components/raftstore/src/store/fsm/store.rs

```diff
@@ -1333,7 +1337,9 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
             }
         }
         workers.coprocessor_host.shutdown();
-        workers.future_poller.shutdown_now().wait().unwrap();
+        workers
+            .future_poller
+            .shutdown_timeout(Duration::from_nanos(0));
```

Comment:
It's different from before. Previously, this waits forever until all tasks finish. Now it doesn't wait. I'm not sure if this will cause any problem.

Reply:
By the way, the new runtime does not provide the equivelent function

Reply:
`servers.node.stop();` will be executed only when tikv needs exit which means `RaftBatchSystem` shutdown when tikv exits. So potential resource leak is OK for my opion. /cc @gengliqi

Reply:
Hope there isn't any test depending on this behavior 😂